### ISSUE TITLE
Crash when playing tagesschau24 on Android 6

### ISSUE THREAD

**Issue by cemrich** [mediathekview/zapp#268]
Stacktrace:

```
2021-12-05 18:59:15.357 3173-3173/de.christinecoenen.code.zapp E/a: ACRA caught a NotFoundException for de.christinecoenen.code.zapp
    android.content.res.Resources$NotFoundException: File res/drawable-xhdpi-v4/channel_logo_tagesschau24.png from xml type xml resource ID #0x7f080072
        at android.content.res.Resources.loadXmlResourceParser(Resources.java:2821)
        at android.content.res.Resources.loadXmlResourceParser(Resources.java:2776)
        at android.content.res.Resources.getXml(Resources.java:1214)
        at s1.h.b(VectorDrawableCompat.java:654)
        at t6.j.b(VideoInfoArtworkExtensions.kt:13)
        at t6.f.o(Player.kt:193)
        at t6.f.a(Player.kt:25)
        at t6.f$f.t(Player.kt:125)
        at n7.a.k(ContinuationImpl.kt:33)
        at e8.y0.run(DispatchedTask.kt:106)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.io.FileNotFoundException: Corrupt XML binary file
        at android.content.res.AssetManager.openXmlAssetNative(Native Method)
        at android.content.res.AssetManager.openXmlBlockAsset(AssetManager.java:485)
        at android.content.res.Resources.loadXmlResourceParser(Resources.java:2803)
        at android.content.res.Resources.loadXmlResourceParser(Resources.java:2776) 
        at android.content.res.Resources.getXml(Resources.java:1214) 
        at s1.h.b(VectorDrawableCompat.java:654) 
        at t6.j.b(VideoInfoArtworkExtensions.kt:13) 
        at t6.f.o(Player.kt:193) 
        at t6.f.a(Player.kt:25) 
        at t6.f$f.t(Player.kt:125) 
        at n7.a.k(ContinuationImpl.kt:33) 
        at e8.y0.run(DispatchedTask.kt:106) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

There might be more channels affected.

**Comment by cemrich** [comment]
Other channels are not affected.

**Comment by cemrich** [comment]
This has been fixed with v6.0.1
